### PR TITLE
Bunch of fixes

### PR DESCRIPTION
**Pull request by Bod9001**
just been looking through staging errors and fixing them

### CODE REVIEW

File: UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs [hunk 1]
```diff
@@ -192,6 +192,8 @@ public float MaxHealth
 
 		protected GameObject LastDamagedBy;
 
+		public List<BodyPart> DismemberingBodyParts = new List<BodyPart>();
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs]: make it [NonSerialized]

File: UnityProject/Assets/Scripts/Items/Implants/Organs/Stomach.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@ namespace HealthV2
 {
 	public class Stomach : BodyPartModification
 	{
-		[NonSerialized] public ReagentContainer StomachContents;
+		public ReagentContainer StomachContents;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Implants/Organs/Stomach.cs:12]: why do we need to serialize the content?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/Stomach.cs:12]: aaah nvm, it is the same thing as the liver container

**Comment by corp-0** [comment]
does this close any recent issue?

**Comment by Bod9001** [comment]
humm, not too sure

**Comment by corp-0** [comment]
would be better to do individual PRs so we can keep track of what it fixes

**Comment by Bod9001** [comment]
this would cover about 6 PRs then

**Comment by corp-0** [comment]
6 PRs is fine. Another option would be at least stating in the PR description what this fixes, I can only guess by looking at the changes.

**Pull request merged**